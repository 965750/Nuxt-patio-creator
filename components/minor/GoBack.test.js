import { describe, test, beforeEach, vitest, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import { createTestingPinia } from '@pinia/testing'
import GoBack from './GoBack.vue'

describe('GoBack', () => {
    let wrapper = null

    beforeEach(() => {
        wrapper = mount(GoBack, {
            shallow: true,
            global: {
                plugins: [
                    createTestingPinia({
                        createSpy: vitest.fn
                    })
                ]
            }
        })
    })

    test('snapshot matches', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})