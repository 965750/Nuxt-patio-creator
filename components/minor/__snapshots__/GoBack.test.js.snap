// Vitest Snapshot v1

exports[`GoBack > snapshot matches 1`] = `
"<nuxtlink data-vitest-back-btn=\\"\\" class=\\"flex backBtn\\" to=\\"/\\" data-v-f542714c=\\"\\"><svg xmlns=\\"http://www.w3.org/2000/svg\\" fill=\\"none\\" viewBox=\\"0 0 24 24\\" stroke-width=\\"1.5\\" stroke=\\"currentColor\\" class=\\"w-6 h-6\\" data-v-f542714c=\\"\\">
    <path stroke-linecap=\\"round\\" stroke-linejoin=\\"round\\" d=\\"M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3\\" data-v-f542714c=\\"\\"></path>
  </svg>
  <p class=\\"ml-2\\" data-v-f542714c=\\"\\">back to project list</p>
</nuxtlink>"
`;
